static/js/hotkey.js

```diff
@@ -606,7 +606,7 @@ exports.process_hotkey = function (e, hotkey) {
             }
             return true;
         case 'n_key':
-            if (overlays.streams_open()) {
+            if (overlays.streams_open() && page_params.can_create_streams) {
                 subs.new_stream_clicked();
             } else {
```

Comment:
I think we want to do nothing if the streams page is open and user can't create streams (i.e. we don't want to narrow fi you're in the streams page under any circumstance)?  So we should put a second if statement inside the first, rather than adding this to the main if statement.

Reply:
@timabbott Amended, thanks for reviewing!